pkg/secrets/strategy.go

```diff
@@ -69,6 +69,13 @@ func (s *Source) UnmarshalRaw(raw map[string]interface{}) error {
 			s.Strategy = k
 			if value, ok := v.(string); ok {
 				s.Hint = value
+
+			} else if s.Strategy == "value" {
+				value, err := unmarshalRawValue(v)
+				if err != nil {
+					return err
+				}
+				s.Hint = value
 			} else {
 				s.Hint = fmt.Sprintf("%v", s.Hint)
```

Comment:
I am a bit confused by this, as the hint should be an empty string at this point?